### PR TITLE
First version of project

### PR DESCRIPTION
**Pull request by Ducatel**
Ref #1

### CODE REVIEW

**Comment by Ducatel** [comment]
@MattMattV review and merge it please ;)

File: hostmydocs/client.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+        try:
+            response = requests.post(request_url, headers=headers, files=files, data=payload, verify=False)
+        except Exception as error:
+            logging.error("Upload fail with error {}".format(error))
+            return False
+        finally:
+            archive_stream.close()
+
+        if response.status_code == 200:
```
Comment (glc-mvion) [inline hostmydocs/client.py]: Prefer use of `requests.codes.ok`

File: hostmydocs/client.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        logging.info("List documentation in HostMyDocs: {}".format(request_url))
+        headers = {"Authorization": 'Basic {}'.format(self.server_config.get_credential_for_basic_auth())}
+
+        try:
+            response = requests.get(request_url, headers=headers, verify=False)
+        except Exception as error:
+            logging.error("Listing fail with error {}".format(error))
+            return None
+
+        if response.status_code != 200:
```
Comment (glc-mvion) [inline hostmydocs/client.py]: Prefer use of `requests.codes.ok`

**Review by glc-mvion — COMMENTED** [review]
Please replace harcoded HTTP status codes with `requests.codes` members

**Comment by glc-mvion** [comment]
LGTM

**Pull request merged**